dist/js/ui-components.js

```diff
@@ -1,2 +1,3810 @@
 t(){this.controller=i,this.require={FonticonPickerCtrl:"^miqFonticonPicker"},this.bindings={title:"@",selector:"@"}}return t}();e.default=o},function(t,e,n){"use strict";Object.defineProperty(e,"__esModule",{value:!0});var i=n(71);e.default=function(t){t.component("miqFonticonFamily",new i.default)}},function(t,e,n){"use strict";Object.defineProperty(e,"__esModule",{value:!0});var i=function(){function t(t,e){this.MiQFonticonService=t,this.$uibModal=e,this.families=[],this.btnClass="btn-default"}return t.$inject=["MiQFonticonService","$uibModal"],t.prototype.addFamily=function(t){this.families.push(t)},t.prototype.openModal=function(){var t=this;void 0===this.fonticons&&(this.fonticons=this.MiQFonticonService.fetch(this.families)),this.toSelect=this.selected,this.modal=this.$uibModal.open({template:n(37),windowClass:"fonticon-modal",keyboard:!1,size:"lg",controllerAs:"$ctrl",controller:["parent",function(t){this.parent=t}],resolve:{parent:function(){return t}}})},t.prototype.closeModal=function(t){t&&(this.selected=this.toSelect),this.modal.close()},t.prototype.markToSelect=function(t){this.toSelect=t},t}();e.FonticonPickerController=i;var o=function(){function t(){this.controller=i,this.template=n(38),this.transclude=!0,this.bindings={btnClass:"@?",selected:"@",inputName:"@"}}return t}();e.default=o},function(t,e,n){"use strict";Object.defineProperty(e,"__esModule",{value:!0});var i=n(73);e.default=function(t){t.component("miqFonticonPicker",new i.default)}},function(t,e,n){"use strict";Object.defineProperty(e,"__esModule",{value:!0});var i=n(74),o=n(72);e.default=function(t){i.default(t),o.default(t)}},function(t,e,n){"use strict";Object.defineProperty(e,"__esModule",{value:!0});var i=n(0),o=function(){function t(){}return t.prototype.fetch=function(e){return i.reduce(e,function(e,n){return e[n.selector]=t.filterRules(n.selector),e},{})},t.filterRules=function(e){return i.chain(document.styleSheets).map(function(t){return t.rules}).map(function(n){return i.filter(n,function(n){return t.isFontIcon(n,e)})}).filter(function(t){return 0!==t.length}).map(function(e){return i.map(e,function(e){return t.clearRule(e.selectorText)})}).flatten().reduce(function(n,i){return n.concat([t.makeRuleObject(e,i)])},[]).value()},t.isFontIcon=function(t,e){return t.selectorText&&1===t.selectorText.indexOf(e)&&-1!==t.cssText.indexOf("content:")},t.clearRule=function(t){return t.replace("::before","").split(",")[0].replace(".","")},t.makeRuleObject=function(t,e){return{class:t+" "+e,selector:"."+t+"."+e}},t}();e.default=o},function(t,e,n){"use strict";Object.defineProperty(e,"__esModule",{value:!0});var i=n(76);e.default=function(t){t.service("MiQFonticonService",i.default)}},function(t,e,n){"use strict";var i=this&&this.__extends||function(){var t=Object.setPrototypeOf||{__proto__:[]}instanceof Array&&function(t,e){t.__proto__=e}||function(t,e){for(var n in e)e.hasOwnProperty(n)&&(t[n]=e[n])};return function(e,n){function i(){this.constructor=e}t(e,n),e.prototype=
```

Comment:
For sure something is wack with dist build, new content isn't concatenated or uglified :-/ :thinking: